src/bin/rav1e.rs

```diff
@@ -206,22 +209,32 @@ fn do_encode<T: Pixel, D: Decoder>(
     y4m_enc.as_mut(),
   )? {
     for frame in frame_info {
-      progress.add_frame(frame);
       if verbose {
-        eprintln!("{} - {}", frame, progress);
+        let frame_text = frame.to_string();
+        progress.add_frame(frame);
+        info!("{} - {}", frame_text, progress);
       } else {
+        progress.add_frame(frame);
         eprint!("\r{}                    ", progress);
       };
     }
 
     output.flush().unwrap();
   }
-  eprint!("\n{}\n", progress.print_summary());
+  if !verbose {
+    eprint!("\r");
+    info!("{}", progress);
+  }
+  progress.print_summary(verbose);
   Ok(())
 }
 
 fn main() {
   better_panic::install();
+  if env::var("RUST_LOG").is_err() {
+    env::set_var("RUST_LOG", "rav1e=info");
```

Comment:
Not sure this is the correct way of doing this

Reply:
You're right, looks like this is: https://docs.rs/env_logger/0.6.2/env_logger/#specifying-defaults-for-environment-variables